sw/device/lib/msg_uart/msg_impl.h

```diff
@@ -0,0 +1,56 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef _MSG_IMPL_H
+#define _MSG_IMPL_H
+
+#include "sw/device/lib/msg_print.h"
+#include "sw/device/lib/uart.h"
+
+// Stringify stuff.
+#define _xstr(a) _str(a)
+#define _str(a) #a
```

Comment:
Give these names that are better scoped to this file, e.g.
```c
#define _MSG_IMPL_STRINGIFY(a) _MSG_IMPL_STRINGIFY(a)
#define _MSG_IMPL_STRINGIFY_INNER(a) #a
```

Reply:
Done.